manifests/init.pp

```diff
@@ -143,7 +142,6 @@
       $default_sshd_gssapicleanupcredentials   = 'yes'
       $default_sshd_acceptenv                  = true
       $default_service_hasstatus               = true
-      $default_sshd_config_serverkeybits       = '1024'
```

Comment:
Needs a conditional here so that if EL7 then set to `undef`, else set to `'1024'`.